### PR TITLE
Improve adaptive auth scripting internals

### PR DESCRIPTION
**Pull request by janakamarasena**

### CODE REVIEW

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java [hunk 1]
```diff
@@ -152,6 +152,7 @@ public JsGraphBuilder createWith(String script) {
                     (LoadExecutor) this::loadLocalLibrary);
             engineBindings.put("exit", (RestrictedFunction) this::exitFunction);
             engineBindings.put("quit", (RestrictedFunction) this::quitFunction);
+            restrictDefaultFunctions(engine);
```
Comment (pulasthi7) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java:155]: shall we have exit and quit functions also under this?

**Review by madurangasiriwardena — APPROVED** [review]

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java (hunk 1 above)
Comment (janakamarasena) [reply components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java]: done

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java [hunk 2]
```diff
@@ -150,8 +160,7 @@ public JsGraphBuilder createWith(String script) {
                     (PromptExecutor) this::addShowPrompt);
             globalBindings.put(FrameworkConstants.JSAttributes.JS_FUNC_LOAD_FUNC_LIB,
                     (LoadExecutor) this::loadLocalLibrary);
-            engineBindings.put("exit", (RestrictedFunction) this::exitFunction);
-            engineBindings.put("quit", (RestrictedFunction) this::quitFunction);
```
Comment (pulasthi7) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java:154]: these functions may not need anymore (exitFunction, quitFunction)

Comment (janakamarasena) [reply components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java:154]: deprecated

**Review by pulasthi7 — APPROVED** [review]

**Pull request merged**